content/en/docs/reference/tools/_index.md

```diff
@@ -29,6 +31,11 @@ to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resou
 [Helm](https://helm.sh/) is a tool for managing packages of pre-configured
 Kubernetes resources. These packages are known as _Helm charts_.
 
+{{< warning >}}
```

Comment:
👋 @RA489 .
What do you think about using the `thirdparty-content` shortcode?
Can you use this shortcode instead of the custom warning message on lines 34-37.
It probably makes sense to place the shortcode right under the Helm heading.

```
{{% thirdparty-content %}}
```

Reply:
I'd put the shortcode immediately after `<!-- body -->` because part of the warning mentions that the following content is sorted alphabetically (and, handily, Helm sorts before Kompose in US English collation).

Reply:
sure I'll update accordingly.

Reply:
>  @RA489 .
> What do you think about using the `thirdparty-content` shortcode?
> Can you use this shortcode instead of the custom warning message on lines 34-37.
> It probably makes sense to place the shortcode right under the Helm heading.
> 
> ```
> {{% thirdparty-content %}}
> ```

updated to use the  {{% thirdparty-content %}}

Reply:
updated.